Makefile

```diff
@@ -477,14 +477,17 @@ pg16-disk-test: docker-build-pg16-disk-test load-pg16-disk-test
 
 ##@ End-to-End tests
 
-.PHONE: e2e-tools
```

Comment:
📞

Reply:
Yeah, we either need a linter or that thing @sharnoff suggests (what is it's name?), I imagine it doesn't have such problems.